packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -742,12 +756,7 @@ export class AtomicSearchBox implements InitializableComponent<Bindings> {
             />
             {this.renderSuggestions()}
           </SearchBoxWrapper>,
-          !this.suggestionManager.suggestions.length && (
```

Comment:
Why are we removing this condition? Does this not change the component's rendering behavior?